v3.1-client.yml

```diff
@@ -692,241 +692,243 @@ definitions:
         x-validation: true
       home_language_code:
         enum:
-        - cpp
-        - kan
-        - slk
-        - yor
-        - guj
-        - ukr
-        - ara
         - jav
-        - mkd
-        - mon
-        - pus
-        - urd
+        - other
```

Comment:
Just making sure, v3.1-client is all auto-generated, right?

Reply:
yes!